push/ups_integration_with_openshift.md

```diff
@@ -0,0 +1,100 @@
+# Unified Push Server Integration with OpenShift 
+
+## Introduction
+
+This proposal outlines some developer experience issues with provisioning, setting up and configuring the Unified Push Server (UPS) on OpenShift, and integrating it with Apps using the Aerogear SDK.
+
+## Problem Description
+
+The Unified Push Server is made up of a Java application & a database (MySQL).
+It also has a dependecy on an auth provider (currently Keycloak) for users of the UPS Administration UI or REST API.
```

Comment:
The REST API is basically two parts
* Management APIs (e.g. everything thats under `/application` or `/metrics` (which really is analytics), that also can be controlled via the UI and its wizard for setting up `PushApplications` and their `Variants
* _public_ APIs (endpoints for the actual mobile devices (E.g. for [registration](https://aerogear.org/docs/specs/aerogear-unifiedpush-rest/index.html#246535932) or any backend-service for [sending](https://aerogear.org/docs/specs/aerogear-unifiedpush-rest/index.html#397083935))

Reply:
Good to know.
I'll update with more specific info